examples/flax/text-classification/README.md

```diff
@@ -83,14 +83,24 @@ We also ran each task once on a single V100 GPU, 8 V100 GPUs, and 8 Cloud v3 TPU
 overall training time below. For comparison we ran Pytorch's [run_glue.py](https://github.com/huggingface/transformers/blob/master/examples/pytorch/text-classification/run_glue.py) on a single GPU (last column).
 
 
-| Task  | 8 TPU   | 8 GPU   | 1 GPU      | 1 GPU (Pytorch) |
-|-------|---------|---------|------------|-----------------|
-| CoLA  |  1m 46s |  1m 26s | 3m 6s      | 4m 6s           |
-| SST-2 |  5m 30s |  6m 28s | 22m 6s     | 34m 37s         |
-| MRPC  |  1m 32s |  1m 14s | 2m 17s     | 2m 56s          |
-| STS-B |  1m 33s |  1m 12s | 2m 11s     | 2m 48s          |
-| QQP   | 24m 40s | 31m 48s | 1h 20m 15s | 2h 54m          |
-| MNLI  | 26m 30s | 33m 55s | 2h 7m 30s  | 3u 7m 6s        |
-| QNLI  |  8m     |  9m 40s | 34m 20s    | 49m 8s          |
-| RTE   |  1m 21s |     55s | 1m 8s      | 1m 16s          |
-| WNLI  |  1m 12s |     48s | 38s        | 36s             |
+| Task  | TPU v3-8  | 8 GPU      | 1 GPU      | 1 GPU (Pytorch) |
```

Comment:
```suggestion
| Task  | TPU v3-8  | 8 GPU  V100    | 1 GPU  V100   | 1 GPU (Pytorch) V100 |
```

Just to make it more visible - think it would be a good idea to add `V100` here as well

Reply:
The table below probably needs some adaption, so fixing in locally might make more sense

Reply:
I think it's a bit too heavy in the table, can we tell it in the description above instead?

Reply:
Okey, then it's fine to leave it as it is I think!